syntax = "proto3";

message RegisterMessage
{
    int32 machineId = 1;
}